### PR TITLE
{chem}[foss/2019b] NAMD v2.13

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
iris-001 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/d05b7002d66c0bc3b36f1de1bc07ab37 for a full test report.

**Comment by ekieffer** [comment]
Test report by @ULHPC
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
iris-001 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/a53e5dfe8b1816034b5d2595ce5717fb for a full test report.

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-001 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/a0e374758e5ca71da6326dd02facee4d for a full test report.

File: easybuild/easyconfigs/n/NAMD/NAMD-2.13-foss-2019b-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name = 'NAMD'
+version = '2.13'
+versionsuffix = '-mpi'
+
+homepage = 'http://www.ks.uiuc.edu/Research/namd/'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/NAMD/NAMD-2.13-foss-2019b-mpi.eb:5]: @ekieffer the tests are failing because they now require `https` even in `homepage`

**Comment by boegel** [comment]
@ekieffer Please ignore that last comment by my bot, it clearly needs work... :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/e129dc5d630068c5cebea35e8bd2f5a7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/831befe1fee2062ce68f8b5a58335246 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-x-1 - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz (haswell), Python 3.6.8
See https://gist.github.com/bb66a3d7d94d58414513c447657283e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2407.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/71f14d32ee63a869b9e3f1e3ed1e138d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ekieffer!

**Comment by boegel** [comment]
Going in, thanks @ekieffer!

**Pull request merged**